tests/testthat/test-multitables.R

```diff
@@ -87,6 +87,29 @@ with_mock_HTTP({
             expect_is(mtable, "Multitable")
         })
     })
+    
+    test_that("importMultitable", {
+        with_POST("https://app.crunch.io/api/datasets/1/multitables/4de322/", {
+            mtable <- newMultitable(~ gender + mymrset, data=ds, name="New multitable")
+            expect_is(mtable, "Multitable")
+        })
+        expect_POST(m <- importMultitable(ds, mtable, name='copied_multitable'),
+                    'https://app.crunch.io/api/datasets/1/multitables/',
+                    '{"element":"shoji:entity","body":{', 
+                    '"multitable":"https://app.crunch.io/api/datasets/1/multitables/4de322/",',
+                    '"name":"copied_multitable"}}')
+    })
+    
+    test_that("multitable show method", {
+        with_POST("https://app.crunch.io/api/datasets/1/multitables/4de322/", {
+            mtable <- newMultitable(~ gender + mymrset, data=ds, name="Shared multitable")
+            expect_is(mtable, "Multitable")
+        })
+        expect_output(mtable, paste(paste0("Multitable ", dQuote("Shared multitable")),
```

Comment:
Where reasonable, I try to keep to an 80-char width, you know, in case punchcards come back. A good place to line break in tests like this is between the object being tested and the expected value, so in this case, between `mtable,` and `paste(paste0(...`